tests/k6tests/stressTest.productInfo.vus1.js

```diff
@@ -0,0 +1,15 @@
+import http from 'k6/http';
+
+export let options = {
+  'vus': 1,
+  'duration': '10s',
+  'thresholds': {
+    'http_req_failed': ['rate<0.01'], // http errors should be less than 1%
+    'http_req_duration': ['p(95)<200'] // 95% of requests should be below 200ms
+  }
+};
+
+export default function () {
+  let resProductInfo = http.get('http://localhost:3001/products/1/');
```

Comment:
for all of your stress tests, you could also include a check to make sure you are receiving the expected data, not just a response

Reply:
Thanks for the tip. I will try that.